### ISSUE TITLE
Future maintenance of splancs

### ISSUE THREAD

**Issue by rsbivand** [rsbivand/legacy_wrap#1]
Hi Barry @barryrowlingson: Re. https://github.com/rsbivand/legacy_wrap/blob/main/splancs.md, might you be willing to consider taking over maintenance of `splancs`? Roger

**Comment by barryrowlingson** [comment]
Is there anything splancs can do that spatstat (or other packages) doesn't do better? The raised incidence code (triblik, tribble) might not be implemented elsewhere but its a fairly simple maximum likelihood expression that anyone with a bit of motivation could write fairly quickly (there's some parameter constraints that need consideration but that's about it).

30 years is a fair run and I'm not sure its worth the effort of maintaining on CRAN. Although maybe the 40 or so dependent packages might think otherwise... hmmm...

**Comment by rsbivand** [comment]
Right, the problem is the reverse dependencies. Hence the question - maybe you can think of others whose packages need `splancs` who could step in?

**Comment by rsbivand** [comment]
@barryrowlingson I put `splancs` on https://github.com/rsbivand/splancs with a pkgdown site too, after being asked to replace `dfloat()` with `dble()` (done). I can submit this version, is that OK? From then on, I'd like to stop maintaining, and transfer the repo.
